djangocms_picture/templates/djangocms_picture/picture.html

```diff
@@ -0,0 +1,77 @@
+ {% load thumbnail %}
+
+{% if instance.get_link %}
```

Comment:
so I read you should define that stuff in the model itself (best practice) but you want me further down to create a template tag anyhow?